jupyter-lib/api/src/main/kotlin/org/jetbrains/kotlinx/jupyter/api/libraries/JupyterIntegration.kt

```diff
@@ -71,10 +71,14 @@ abstract class JupyterIntegration : LibraryDefinitionProducer {
         }
 
         inline fun <reified T : Any> render(noinline renderer: CodeCell.(T) -> Any) {
-            val execution = ResultHandlerExecution { _, property ->
+            return renderWithHost { _, value: T -> renderer(this, value) }
+        }
+
+        inline fun <reified T : Any> renderWithHost(noinline renderer: CodeCell.(ExecutionHost, T) -> Any) {
```

Comment:
Just got that it is not what I need for #183. This function renders property; however, I need to render an arbitrary object.